submissions/dmytro-botnarenko/html-css-popup/index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Document</title>
+</head>
+    <body>
+        <div class="wrapper">
+            <header class="header">
+            <nav>
+                <ul class="navigation__menu">
+                    <li class="navigation__list">
+                        <a href="#" class="navigation__link">Mail</a>
+                    </li>
+                    <li class="navigation__list">
+                        <a href="#" class="navigation__link">Images</a>
+                    </li>
+                    <li class="navigation__list">                
+                        <input id="toggle" class="toggle" type="checkbox">
+                        <label for="toggle" class="navigation__link"><img class="navigation__image" src="img/menu.svg" alt="Go to popup menu"></label>
```

Comment:
Thank you for this useful link. This is not the first article in which I read about the `alt` text.)

In the case of a specific line (23), I can only assume that it is maybe better to write *"Open menu with other links"* or *"Open menu with additional links"* instead of a bit abstract *"Go to popup menu"*. Does this wording fits better in this situation? For now, I'll make a correction and replace with *"Open menu with additional links"*. All other `alt` text in main navigation strip now has same pattern *"Open ..."*

As for the `alt` texts for the icons in the popup menu itself, this is not the first time I have come across that you **may** not to write `alt` text to them if there is a text signature in the link. I've added `alt` text to the icons in Popup menu, but maybe they are redundant. But [this article](https://axesslab.com/alt-texts/) convinced me, and I decided to remove the `alt` texts, since there is the necessary similar text in the `a` links (containers) of the images.

I hope I have interpreted these standards correctly now.

Reply:
Yeah, right decision 👍